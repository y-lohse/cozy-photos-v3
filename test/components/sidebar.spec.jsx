'use strict'

/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'

import Sidebar from '../../src/components/Sidebar'

describe('Sidebar component', () => {
  it('should be rendered correctly', () => {
    const component = shallow(
      <Sidebar />
    ).node
    expect(component).toMatchSnapshot()
  })
})
