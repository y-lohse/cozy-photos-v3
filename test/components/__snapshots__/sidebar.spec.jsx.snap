exports[`Sidebar component should be rendered correctly 1`] = `
<aside
  className="coz-sidebar">
  <_translate />
</aside>
`;
